park-dashboard/src/Pages/ParkInfo/Carousel.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import Carousel from "react-bootstrap/Carousel";
+import "bootstrap/dist/css/bootstrap.css";
+
+function PhotoCarousel(props) {
+  const [images, setImages] = useState([]);
+
+  useEffect(() => {
+    fetchImages();
+  }, []);
+
+  const fetchImages = () => {
```

Comment:
I would suggest adding try catch blocks, in case if API fetching is failed, you would be able to get the error message.

Reply:
Addressed your feedback and added try catch to the network call.